Libraries/LibGUI/GFilePicker.h

```diff
@@ -15,10 +15,10 @@ class GFilePicker final : public GDialog {
     };
 
     static Optional<String> get_open_filepath();
-    static Optional<String> get_save_filepath();
+    static Optional<String> get_save_filepath(const String& m_file_extension);
```

Comment:
The `m_` prefix is for member variables, so please don't use it on an argument name.